tests/simpletest.py

```diff
@@ -106,7 +106,7 @@ def test_invalid_hash_seeds_type(self):
         self.assertRaises(ValueError, pybloomfilter.BloomFilter, 100, 0.05, hash_seeds=[1, 2, 3, 4])
 
     def test_invalid_hash_seeds_typecode(self):
-        self.assertRaises(ValueError, pybloomfilter.BloomFilter, 100, 0.05, hash_seeds=array.array("Q", [1, 2, 3, 4]))
+        self.assertRaises(ValueError, pybloomfilter.BloomFilter, 100, 0.05, hash_seeds=array.array("f", [1, 2, 3, 4]))
```

Comment:
Why floats?

Reply:
NM... I just read the name of the test

Reply:
It's just the first it came to my mind. I didn't realize `Q` is not available in py2.7